src/internal.c

```diff
@@ -2314,6 +2314,12 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
             ctx->CBIORecvFrom = uIPRecvFrom;
         }
         #endif
+    #elif defined(ARDUINO)
+        #if defined(__AVR__)
```

Comment:
Why need this at all? Seems like requiring WOLFSSL_USER_IO is the better option. Then you can remove this.

Reply:
oh, nice. excellent suggestion, thank you. changed.